Task2/aga/TicTacToe/TicTacToe/TheGame.xaml

```diff
@@ -0,0 +1,42 @@
+﻿<Window x:Class="TicTacToe.TheGame"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:local="clr-namespace:TicTacToe"
+        mc:Ignorable="d"
+        Title="The Game" Height="270" Width="300">
+    <Grid Margin="10">
+        <Grid.RowDefinitions>
+            <RowDefinition Height="50"/>
```

Comment:
You could achieve the same chocolate bar look of the board (with spaces between buttons) with putting margin on a Button. See how Styles work in WPF, you could define margin for a button at the top of a page the same way you define CSS in HTML. Google 'WPF Style TargetType'